cob_cartesian_controller/src/cartesian_controller_utils.cpp

```diff
@@ -78,7 +78,7 @@ void CartesianControllerUtils::transformPose(const std::string source_frame, con
         try
         {
             stamped_in.header.stamp = ros::Time(0);
```

Comment:
Both of them are working fine, but I prefer the ros::Time::now() variant since it's more logical to get the transformation with the current time stamp.